client/room.html

```diff
@@ -206,27 +206,36 @@ <h1>Share link</h1>
+      <h2>Copyright Attribution</h2>
+      <p>The default card deck was created by Adrian Kennard and released under CC0 Public Domain license: <a href="https://www.me.uk/cards/">https://www.me.uk/cards/</a>.</p>
+      <p>The copyright attribution information for all games and assets in the public library is available inside the "Variant" notes section of each game. All public library tutorials are released under the CC0 license. Detailed information about the various Creative Commons licenses referred to by abbreviations in variant notes can be found on the <a href="https://creativecommons.org/about/cclicenses/">Creative Commons webpage</a> or at the following links:</p>
+      <ul>
+      <li><a href="https://creativecommons.org/licenses/by/4.0/">CC BY</a></li>
+      <li><a href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA</a></li>
+      <li><a href="https://creativecommons.org/licenses/by-nc/4.0/">CC BY-NC</a></li>
+      <li><a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">CC BY-NC-SA</a></li>
+      <li><a href="https://creativecommons.org/licenses/by-nd/4.0/">CC BY-ND</a></li>
+      <li><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">CC BY-NC-ND</a></li>
+      <li><a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0 (Public Domain)</a></li>
+      </ul>
+      <h2>Online Gameplay</h2>
+      <p>This is an open-source, free to use platform.</p>
+      <p>Anyone that knows the URL of the room that you are using may come in to the room at any time and take any action, including playing or changing games in the room or deleting games. Keep a personal backup of everything that you care about.</p>
+      <p>Know who you are playing with.  The platform is unmoderated and no one controls what may happen. Anything you do or share is your own responsibility.</p>
+      <p>Cheating is trivially easy, unpreventable, and undetectable.  Just play for fun.</p>
+      <p>If you are under the age of majority in the location you are from, be sure to comply with the rules for accessing online material that apply to you.</p>
+      <h2>Privacy Policy</h2>
+      <p>VirtualTabletop.io does not ask for or collect any personal identification information.</p>
+      <p>VirtualTabletop.io does not use cookies.</p>
+      <p>VirtualTabletop.io does, however, collect the "player name" and keeps that in local storage on the user's device.  The player name defaults to a number, but is customizable to any user entered data. VirtualTabletop.io does not analyze or use that information except to display it on the screen while accessing the website.</p>
     </div>
 
-    <div id="internalErrorOverlay" class="overlay">
-      <h1>Internal Error</h1>
```

Comment:
These two overlays should not be removed.

Reply:
yeah I copied it over the wrong </div>